containers/jetty/src/main/java/com/yahoo/athenz/container/AthenzJettyContainer.java

```diff
@@ -178,6 +179,46 @@ public void addServletHandlers(String serverHostName) {
             rewriteHandler.addRule(disableKeepAliveRule);
         }
         
+        // Security: Respond the "Expect-CT: max-age=31536000, report-uri=..." header
+
+        HeaderPatternRule expectCtRule = new HeaderPatternRule();
+        expectCtRule.setPattern("/*");
+        expectCtRule.setName("Expect-CT");
+        expectCtRule.setValue("max-age=31536000, report-uri=\"http://csp.yahoo.com/beacon/csp?src=yahoocom-expect-ct-report-only\"");
```

Comment:
This can't be hard coded here. Needs to be extensible for other organizations.

Reply:
Right.
Fixed